### PR TITLE
refactor: live tail and health server check

### PR DESCRIPTION
**Pull request by J1za**
## Description
Delete live tail.  Сheck health server by using `RetryScheduler`
## Linear Ticket
https://linear.app/autokitteh/issue/UI-541/live-tail-webui-and-vscode-extension-behavior
## What type of PR is this? (check all applicable)

- [ ] 💡 (feat) - A New Feature (non-breaking change which adds functionality)
- [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
- [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
- [ ] 🏎 (perf) - Optimization
- [ ] 📄 (docs) - Documentation - Documentation only changes
- [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
- [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white spaces, formatting, missing semi-colons, etc)
- [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
- [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
- [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

## Code Standards
- [ ] Notify only when user attention is needed, otherwise log to console.
  - [ ] Notifications: short description with context identifiers (e.g. id of the manipulated entity).
  - [ ] Logs: full description with full context identifiers (e.g. deploymentId and the relevant projectId).
- [ ] If you're not sure what is the proper behavior, consult with the product team.
- [ ] Reduce the use of `else` by employing early returns to make code more readable and less nested.
- [ ] MVC Separation: Ensure separation of concerns; views should only be manipulated by controllers (also the computing - sorting, fitlering, etc.), not by services, to maintain a clean MVC architecture.
- [ ] Before implementing custom logic, check if existing functions or utilities (like lodash) offer a simpler solution.
- [ ] Avoid using `!important` in CSS where possible.
- [ ] Use memoization for class calculations.
- [ ] Place constant strings in your code using I18n.

### CODE REVIEW

File: src/controllers/project.controller.ts [hunk 1]
```diff
@@ -142,26 +138,20 @@ export class ProjectController {
 		this.fetchSessions();
 	};
 
-	public reconnect = () => {
-		this.deployments = undefined;
-		this.loadAndDisplayDeployments(false);
-		this.fetchSessions();
-	};
-
```
Comment (RonenMars) [inline src/controllers/project.controller.ts:150]: We need this - it runs when we click re-connect in the project UI after the server disconnects.

**Comment by RonenMars** [comment]
LGTM, but please remove `react-toggle` because we are not using it anywhere else in the project.

File: src/controllers/project.controller.ts (hunk 1 above)
Comment (J1za) [reply src/controllers/project.controller.ts:150]: added reconnect

File: src/controllers/project.controller.ts [hunk 2]
```diff
@@ -142,26 +139,30 @@ export class ProjectController {
...
 		this.sessionLogRetryScheduler?.stopTimers();
-		this.deploymentsRetryScheduler?.stopTimers();
+		this.serverHealthMonitorScheduler?.stopTimers();
 		this.deployments = undefined;
 		this.sessions = undefined;
 		this.hasDisplayedError = new Map();
 	};
 
 	public tryToReenable = async () => {
+		console.log("recconect");
```
Comment (RonenMars) [inline src/controllers/project.controller.ts:160]: please remove

File: src/controllers/project.controller.ts [hunk 3]
```diff
@@ -142,26 +139,30 @@ export class ProjectController {
...
-		this.deployments = undefined;
-		this.loadAndDisplayDeployments(false);
-		this.fetchSessions();
+	public reconnect = async () => {
+		await this.checkServerHealth(true);
+
+		if (!this.serverHealthCheckFailed) {
+			this.deployments = undefined;
+			this.loadAndDisplayDeployments();
+		}
```
Comment (RonenMars) [inline src/controllers/project.controller.ts:148]:
```suggestion
		const isServerHealthy = await this.checkServerHealth(true);

		if (isServerHealthy) {
			this.deployments = undefined;
			this.loadAndDisplayDeployments();
		}
```

File: src/controllers/project.controller.ts (hunk 2 above)
Comment (J1za) [reply src/controllers/project.controller.ts:160]: removed

File: src/controllers/project.controller.ts (hunk 3 above)
Comment (J1za) [reply src/controllers/project.controller.ts:148]: changed

**Review by RonenMars — APPROVED** [review]

**Comment by RonenMars** [comment]
Looks good, last comments:
1. Please remove `react-toggle` because we are not using it anywhere else in the project.
2. Commit both - modified package.json and updated package-lock.json.

**Pull request merged**